raiden/smart_contracts/NettingChannelLibrary.sol

```diff
@@ -282,15 +295,47 @@ library NettingChannelLibrary {
             throw;
         }
 
+        uint64 nonce;
         assembly {
-            nonce := mload(add(transfer_raw, 12))  // skip cmdid and padding
+            nonce := mload(add(transfer, 12))  // skip cmdid and padding
         }
-
         if (nonce < sender.nonce || nonce == sender.nonce) {
             throw;
         }
-
         decodeAndAssign(sender, transfer_raw);
+    }
+
+    /// @notice updateTransfer Updates (disputes) the state after closing.
+    /// @param caller_address The counterparty to the channel. The participant
+    ///                       that did not close the channel.
+    /// @param first_transfer The transfer the counterparty believes is the
+    ///                        valid state for the first participant.
+    /// @param second_transfer The transfer the counterparty believes is the
+    ///                        valid state for the second participant.
```

Comment:
Note: I'm unsure, if there shall be any case, where a participant may be allowed to and interested in updating/disputing with a transfer of their own. I don't believe this is a good approach, but I may be convinced otherwise.

For clarity, please update the variables and docs so that `first`/`second` become `our` and `their` (in the intended direction).